### DISCUSSION TITLE
Deploy static HTML

### DISCUSSION THREAD

**Discussion by milanpanin** [Dokploy/dokploy#471] [Q&A] [answered]
I've been trying to deploy a simple indes.html for an hour, but without success. Can you explain to me what I'm doing wrong? static.zip contain only index.html

![Capture](https://github.com/user-attachments/assets/74c05fa3-5b6a-4143-8b21-82417c7c1340)
![Screenshot 2024-09-20 at 19-41-46 Dokploy](https://github.com/user-attachments/assets/b41d964e-6843-4f0b-9367-f6813423d99d)

**Comment by Siumauricio** [comment]
Can you show the domains tab?

**Reply by milanpanin** [reply]
![image](https://github.com/user-attachments/assets/7bd65908-d35d-42c6-b789-bf42360f2ff3)

**Reply by Siumauricio** [reply, marked as answer]
Change the port to 80
